doc/src/contributing/build-docs.rst

```diff
@@ -49,7 +65,9 @@ For Fedora (and maybe other RPM-based distributions), install the
 prerequisites::
 
    dnf install python3-sphinx librsvg2 ImageMagick docbook2X texlive-dvipng-bin
```

Comment:
The space between lines is to facilitate users to copy the commands individually for the terminal easily as on the website it will be displayed as having gaps between each other, which will enable easier selection of required commands. 
As for the case of leading spaces, it will not appear on the website, and thus will not affect lines entered by users in terminal. It is only for the purpose of helping people understand the build-docs.rst file easily as it is easier to understand that the indented part will be displayed in a box in the final page, while the non-indented part will be displayed as plain text.